### PR TITLE
Correct grammatical errors in README.md

### PR DESCRIPTION
**Pull request by alienishi**
# PR Summary

I have corrected several grammatical errors in README.md.

## PR Context

Since README.md is like an instruction manual it should be error free.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by alienishi** [comment]
@microsoft-github-policy-service agree

**Comment by alienishi** [comment]
@iSazonov 
Please review my PR and if according to the guidelines please do merge it :)

**Comment by alienishi** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by daxian-dbw** [comment]
@sdwheeler Can you please review the changes in this PR? Thanks!

File: README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
 
 Welcome to the PowerShell GitHub Community!
 [PowerShell](https://learn.microsoft.com/powershell/scripting/overview) is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized
-for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models.
-It includes a command-line shell, an associated scripting language and a framework for processing cmdlets.
+for dealing with structured data (e.g., JSON, CSV, XML, etc.), REST APIs, and object models.
```
Comment (sdwheeler) [inline README.md:5]:
```suggestion
for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models.
```

File: README.md [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
+It includes a command-line shell, an associated scripting language, and a framework for processing cmdlets.
 
 [logo]: https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/ps_black_64.svg?sanitize=true
 
 ## Windows PowerShell vs. PowerShell Core
 
-Although this repository started as a fork of the Windows PowerShell codebase, changes made in this repository do not make their way back to Windows PowerShell 5.1 automatically.
+Although this repository started as a fork of the Windows PowerShell codebase, changes made in this repository do not automatically make their way back to Windows PowerShell 5.1.
```
Comment (sdwheeler) [inline README.md:12]:
```suggestion
Although this repository started as a fork of the Windows PowerShell codebase, changes made in this repository are not automatically ported back to Windows PowerShell 5.1.
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
I have a couple of minor suggestions.

**Comment by alienishi** [comment]
Hi @sdwheeler 
Thanks for the suggestions and correcting me out 
I have made the changes :)

**Review by sdwheeler — APPROVED** [review]
LGTM

**Comment by alienishi** [comment]
@iSazonov @SteveL-MSFT 
Please review my PR and if according to the guidelines please do merge it :)

**Review by iSazonov — APPROVED** [review]

**Pull request merged**